clusters/build-clusters/01_cluster/ci/_origin-release-build/golang-1.20/Dockerfile

```diff
@@ -0,0 +1,50 @@
+#
+# This is the image that controls the standard build environment for releasing OpenShift Origin.
+# It is responsible for performing a cross platform build of OpenShift.
+#
+# The standard name for this image is openshift/origin-release
+#
+FROM quay.io/centos/centos:stream9
+
+ENV VERSION=1.20.7 \
+    GOCACHE=/go/.cache \
+    GOARM=5 \
+    GOPATH=/go \
+    GOROOT=/usr/local/go \
+    LOGNAME=deadbeef
+
+ENV PATH=$PATH:$GOROOT/bin:$GOPATH/bin
+
+COPY cbs-paas7-openshift-multiarch-el7-build.repo /etc/yum.repos.d/
+RUN yum install -y epel-release && \
+    rpm -V epel-release && \
+    INSTALL_PKGS="bc bind-utils bsdtar bzr ceph-common createrepo device-mapper device-mapper-persistent-data e2fsprogs ethtool file findutils gcc git glibc-static glib2-devel gpgme gpgme-devel hostname iptables jq krb5-devel libassuan libassuan-devel libseccomp-devel libvirt-devel lsof make mercurial nmap-ncat openssl protobuf-compiler protobuf-devel rsync socat systemd-devel sysvinit-tools tar tito tree util-linux wget which xfsprogs zip goversioninfo" && \
+    yum install -y $INSTALL_PKGS && \
+    rpm -V $INSTALL_PKGS && \
+    yum clean all && \
+    touch /os-build-image && \
+    git config --system user.name origin-release-container && \
+    git config --system user.email origin-release@redhat.com
+
+RUN \
+    set -x &&  \
+    curl --fail --retry 3 -L https://golang.org/dl/go$VERSION.linux-amd64.tar.gz | tar -C /usr/local -xzf - && \
+    go install golang.org/x/tools/cmd/cover@latest \
+        golang.org/x/tools/cmd/goimports@latest && \
+    go install golang.org/x/lint/golint@latest && \
+    go install github.com/openshift/release/tools/gotest2junit@latest && \
+    go install github.com/openshift/imagebuilder/cmd/imagebuilder@latest && \
+    go install gotest.tools/gotestsum@latest && \
+    mv $GOPATH/bin/* /usr/bin/ && \
+    rm -rf $GOPATH/* $GOPATH/.cache && \
+    mkdir $GOPATH/bin && \
+    ln -s /usr/bin/imagebuilder $GOPATH/bin/imagebuilder && \
+    ln -s /usr/bin/goimports $GOPATH/bin/goimports
+    # TODO: symlink for backwards compatibility with hack/build-images.sh only, remove
```

Comment:
Of note, when comparing with https://github.com/openshift/release/pull/35192/files#diff-0c04913b786701cb949fef004edddb9dd5012aa8e3eda40bcc3c5adee8860007R46-R54 I have removed `dep` and `-mod=vendor` as they are no longer needed in my opinion.